src/reporters/json.ts

```diff
@@ -472,9 +472,9 @@ export default class JSONReporter extends BaseReporter {
             this.writeJSON({
               type: 'journey/filmstrips',
               journey,
-              payload: {
-                index,
-                start: strip.start,
+              payload: { index },
+              root_fields: {
+                browser: { relative_trace: { start: strip.start } },
```

Comment:
Can we add units to `start` as well?

Reply:
They are already relative right? Do you have any suggestion?

Reply:
start.time.<unit>?

Reply:
Yeah, that's what I was thinking with the `us`

Reply:
Done.